wiki/Performance_points/en.md

```diff
@@ -132,3 +112,39 @@ Total score is still important to individual map rankings, and this may produce
 No system is completely perfect, and performance point totals will certainly vary between mapsets and certain mod combinations, even when the subjective difficulty of those plays may be lower than a more difficult map.
 
 Overall, the current performance points system has been engineered to be as fair as possible under the constraints of its model.
+
+## History
+
+The initial implementation of performance points was revealed to the public in April 2012 and was then known only as the mysterious *"???"* project.
+
+With the full name revealed later on in the month, this new system sought to change the previous standard of player performance from simply total [score](/wiki/Gameplay/Score) to something that accurately reflected skill. The new system was met with widespread acclaim among the player base at the time.
+
+Several months after its reveal, the 20120722-24 osu! release officially implemented the system to fully replace the old [Ranked](/wiki/Beatmap/Category#ranked) score system, with new scores being calculated every 30 minutes. Later on, in August of the same year, the system was improved to update in real-time.
+
+It continued to exist in this capacity for more than a year of service until [Tom94](https://osu.ppy.sh/users/1857058), the creator of the *osu!tp* scoring metric, joined the [osu! team](/wiki/People/osu!_team) and implemented his design into the system. The resulting system was titled *ppv2*, and became live on January 27, 2014, therefore renaming the old system to *[ppv1](/wiki/Performance_points/ppv1)*. Since then, the system has been continuously maintained. but no major updates have been made.
```

Comment:
wrong tense

```suggestion
It continued to exist in this capacity for more than a year of service until [Tom94](https://osu.ppy.sh/users/1857058), the creator of the *osu!tp* scoring metric, joined the [osu! team](/wiki/People/osu!_team) and implemented his design into the system. The resulting system was titled *ppv2*, and became live on January 27, 2014, therefore renaming the old system to *[ppv1](/wiki/Performance_points/ppv1)*. Since then, the system was continuously maintained. but no major updates were made.
```